sql/core/src/main/scala/org/apache/spark/sql/expressions/WindowSpec.scala

```diff
@@ -174,28 +191,22 @@ class WindowSpec private[sql](
    */
   // Note: when updating the doc for this method, also update Window.rangeBetween.
   def rangeBetween(start: Long, end: Long): WindowSpec = {
```

Comment:
We should also create API's that allow for other types of literals, at least one for `CalendarInterval`s.